packages/appium/docs/en/ecosystem/index.md

```diff
@@ -53,3 +56,15 @@ the Appium client depends on) since that is what you will use as your primary in
 
 In general, any W3C WebDriver spec-compatible client will also integrate well with Appium, though
 some Appium-specific commands may not be implemented in other clients.
+
+## Plugins
+
+Appium Plugins offer ways to extend or modify Appium's behaviour. This is the list of plugins that
+are currently recognized by Appium:
+
+|Plugin|Installation Key|Description|Support|
+|---|---|---|---|
+|[Images](https://github.com/appium/appium/tree/master/packages/images-plugin)|`images`|Image matching and comparison features|Appium Team|
+|[Execute Driver](https://github.com/appium/appium/tree/master/packages/execute-driver-plugin)|`execute-driver`|Run entire batches of commands in a single call to the Appium server|Appium Team|
+|[Relaxed Caps](https://github.com/appium/appium/tree/master/packages/relaxed-caps-plugin)|`relaxed-caps`|Relax Appium's requirement for vendor prefixes on capabilities|Appium Team|
+|[AltUnity](https://github.com/headspinio/appium-altunity-plugin)|`altunity`|Target Unity games and apps for automation with a new context, via the AltUnityTester framework|HeadSpin|
```

Comment:
https://github.com/appium/appium#plugins-maintained-by-the-appium-team
[universal-xml](https://github.com/appium/appium/tree/master/packages/universal-xml-plugin) as well?